### DISCUSSION TITLE
How to navigate between tabs?

### DISCUSSION THREAD

**Discussion by johnchase** [nvim-tree/nvim-tree.lua#2747] [Q&A]
I have just started using nvim-tree and it is working great, thanks for the work on it! 

I'm sure this is something simple I am missing, however, I cannot figure out how to navigate between tabs, or close a tab without the mouse. Perhaps someone can point me in the right direction, I have not seen this in an issue or documentation. Thanks

**Comment by johnchase** [comment]
Got it,  I was using [LunarVim](https://github.com/ChristianChiarulli/LunarVim) which had set keymappings for the buffers, so not an issue for nvim-tree at all.

**Comment by pgrepds** [comment]
@johnchase May I ask *how* this is working in LunarVim? I haven't found any hints in the documentation.

**Comment by alcaitiff** [comment]
shift+h to previous and shift+l to next.

**Comment by manasomali** [comment]
@alcaitiff Thanks, How did you found this information?

**Comment by Mkamvi** [comment]
try "**_:bnext_**" and ":_**bprev**_"

<img width="560" alt="截屏2022-08-28 14 21 02" src="https://user-images.githubusercontent.com/52448047/187060622-887de887-294b-4740-b34e-622d40d6ad35.png">

**Comment by longguzzz** [comment]
try `<leader>bb` and `<leader>bn`

**Comment by hoodielive** [comment]
Thank you guys! Special thanks to @longguzzz. I have been wracking my brain all day over this! 😊

**Comment by scotttesler** [comment]
Additionally, `<leader>bj` displays letters on each tab that, if pressed, jumps you to that tab.

By the way, `<leader>bh` closes all tabs to the left of your current tab, so… be careful 😅

**Comment by WakeUpHugo** [comment]
> Additionally, `<leader>bj` displays letters on each tab that, if pressed, jumps you to that tab.
> 
> By the way, `<leader>bh` closes all tabs to the left of your current tab, so… be careful 😅

OMG this is beautiful. New to neovim. Thank you for this info.

**Comment by perfectmak** [comment]
Also, I'm just setting up nvim and I found that `:bprev` and `:bnext` worked. But also simply pressing `tab` and `shift+tab` also worked.
